openghg/standardise/flux_timeseries/_crf.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+from pathlib import Path
+from typing import Dict, Optional, Union
+from openghg.store import infer_date_range
+
+
+def parse_crf(
+    filepath: Path,
+    species: str,
+    source: str = "crf",
```

Comment:
The `source` input is included to describe where the emissions are coming from. In the case of the CRF data this may be best described as anthropogenic so a more intuitive default could be something like: "anthro".

Reply:
updated to anthro